packages/wonder-blocks-core/src/components/render-state-root.js

```diff
@@ -6,8 +6,9 @@ import * as React from "react";
 // https://github.com/import-js/eslint-plugin-import/issues/2073
 // eslint-disable-next-line import/named
 import {RenderState, RenderStateContext} from "./render-state-context.js";
+import {useRenderState} from "../hooks/use-render-state.js";
```

Comment:
I updated existing uses of `useContext(RenderStateContext)` which should hopefully save some bytes.